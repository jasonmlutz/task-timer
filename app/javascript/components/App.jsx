import React from 'react';
import { useRoutes } from 'react-router-dom';

import NewUserForm from './Users/NewUserForm';

function App() {
  const routes = useRoutes([
    { path: '/', element: <NewUserForm /> },
  ]);
  return (
    <div>{routes}</div>
  );
}

export default App;
