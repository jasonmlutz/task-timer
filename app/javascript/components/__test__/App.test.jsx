/**
 * @jest-environment jsdom
 */
// REACT
import React from 'react';
import { BrowserRouter } from 'react-router-dom';
// TEST UTILITIES
import renderer from 'react-test-renderer';

// COMPONENTS
import App from '../App';

describe('App component', () => {
  describe('STATIC TESTS', () => {
    test('renders correctly from snapshot', () => {
      const tree = renderer.create(
        <BrowserRouter>
          <App />
        </BrowserRouter>,
      ).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
