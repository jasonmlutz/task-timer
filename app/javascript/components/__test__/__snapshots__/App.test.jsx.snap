// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component STATIC TESTS renders correctly from snapshot 1`] = `<div />`;
